app.py

```diff
@@ -59,25 +59,18 @@ def add_pet_form():
 @app.route('/<int:pet_id>/', methods=['GET', 'POST'])
 def edit_pet_form(pet_id):
     pet = get_pet_by_id(pet_id)
-    form = PetForm(obj=pet)
+    form = EditPetForm(obj=pet)
+    
 
     if form.validate_on_submit():
-        name = form.name.data
-        species = form.species.data
-        photo_url = form.photo_url.data
-        age = form.age.data
-        notes = form.notes.data
+        pet.photo_url = form.photo_url.data
```

Comment:
You can do this here, no need for https://github.com/coachdaveofficial/wtforms-exercise/pull/1/files#diff-568470d013cd12e4f388206520da39ab9a4e4c3c6b95846cbc281abc1ba3c959R70-R71
```suggestion
        pet.photo_url = form.photo_url.data or None
```